src/components/mastra/templates/TemplateCard.tsx

```
@@ -0,0 +1,124 @@
+            <span className="mr-1">{complexityIcons[template.metadata.complexity]}</span>
+            <span className="capitalize">{template.metadata.complexity}</span>
+          </div>
+        )}
+        {template.metadata?.author && (
+          <div className="text-xs text-gray-500">by {template.metadata.author}</div>
+        )}
+      </div>
+
+      {selected && (
+        <div className="mt-3 p-2 bg-blue-100 border border-blue-300 rounded text-xs text-blue-800">
+          ✓ Selected - Click "Load Template" to continue
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)